src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -159,6 +159,7 @@ def repo_status_icon(status)
 
     description = REPO_STATUS_DESCRIPTIONS[status] || 'Unknown state of repository'
     description = 'State needs recalculations, former state was: ' + description if outdated
+    description += details if details
 
```

Comment:
you can't just add a string to that without any further context. Your result will be "format state was: scheduledpublisher broken"